tests/onnx/quantization/test_classification_models_graph.py

```diff
@@ -51,6 +51,6 @@ def test_min_max_quantization_graph(tmp_path, model_to_test, model):
     torch.onnx.export(model, x, onnx_model_path, opset_version=13)
 
     original_model = onnx.load(onnx_model_path)
-    quantized_model = min_max_quantize_model(model_to_test.input_shape, original_model)
+    quantized_model = ptq_quantize_model(model_to_test.input_shape, original_model)
```

Comment:
Take a look the impact of changes `min_max_quantize_model  `to `ptq_quantize_model`.
![image](https://user-images.githubusercontent.com/32935044/194572457-7fdf0c2f-6186-4b2d-b611-2b6f9db59604.png)
![image](https://user-images.githubusercontent.com/32935044/194572496-73fa3dde-a0ef-4a6b-b3c0-67dc5c344e80.png)
For the graph tests I recommend to revert these changes.

Reply:
And to be confident that graphs are equal in ptq and minmax we should return back the corresponding test

Reply:
What is wrong with the tests? Now, they are the most complete and honest with the `ptq_quantize_model` method instead of the `min_max_quantize_model`. 
If you are talking about the graph checks, it would be better to change the tests and add only the quantizers insertion step without statistics collection and quantization as is.
If you are talking about time consumption, I don't know how to comment on this.

Reply:
The originally these tests are aimed to check the quantization placements with the reference. 
I agree with you that the collection statistics step is not necessary here, could you update them accordingly?

Reply:
I think we should update the whole of the post-training tests in a separate PR because of the size of the needed updates.
By the way, the future PR with the MinMax update would be perfect for this.